python/experiments/generate-box-tests-reports.main.kts

```diff
@@ -43,7 +43,7 @@ val testResults = getTestResults(Paths.get("python/box.tests/build/test-results/
 testResults.writeFailedTestsSummary(targetFailedTestsReportPath)
 testResults.writeSummaryTsvToFile(targetBoxTestsReportPath)
 testResults.writeFailureCount(failureCountReportPath)
-generateGitHistoryPlot(gitHistoryPlotPath)
+//generateGitHistoryPlot(gitHistoryPlotPath)  // todo: resolve OOM
```

Comment:
Oddily, works on my machine, so it would be nice if you could include it in this PR:
[git-history-plot.zip](https://github.com/krzema12/kotlin-python/files/7574611/git-history-plot.zip)

FTR, I'll fix it in #50.

Reply:
Done, included. In the future, feel free to commit to the branch yourself if you want ;)

Yeah, the last time that error didn't happen on your machine too... That's strange because I believe mine is quite beefy